ProjectSettings/TagManager.asset

```diff
@@ -6,6 +6,7 @@ TagManager:
   tags:
   - Item
   - Holdable
+  - Wood
```

Comment:
Maybe i'm missing something but I don't think the wood tag is necessary anywhere, so it can be removed?